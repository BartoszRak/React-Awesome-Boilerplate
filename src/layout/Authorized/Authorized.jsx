import React from 'react'
import PropTypes from 'prop-types'
import { withStyles } from '@material-ui/core/styles'

import AuthorizedPages from '~pages/AuthorizedPages'

import styles from './Authorized.style'

export function Authorized({ classes }) {
  return (
    <div className={classes.root}>
      Authorized
      <AuthorizedPages />
    </div>
  )
}

Authorized.propTypes = {
  classes: PropTypes.object,
}

export default withStyles(styles)(Authorized)
