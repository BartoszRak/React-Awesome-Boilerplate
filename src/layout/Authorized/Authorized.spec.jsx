import React from 'react'
import { shallow } from 'enzyme'
import { mocks } from '~utils/tests'

import { Authorized } from './Authorized'

const initComponent = overrides => {
  const mockProps = {
    classes: mocks.classesProxy,
  }
  const mockMethods = {}
  const wrapper = shallow(<Authorized {...mockProps} {...mockMethods} {...overrides} />)
  return { mockProps, wrapper }
}

describe('Authorized: ', () => {
  it('renders without crashing', () => {
    const { wrapper } = initComponent()
    expect(wrapper).toBeTruthy()
  })
  it('should render as expected', () => {
    const { wrapper } = initComponent()
    expect(wrapper).toMatchSnapshot()
  })
})
