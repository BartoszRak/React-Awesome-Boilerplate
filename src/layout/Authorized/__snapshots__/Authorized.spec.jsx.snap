// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authorized:  should render as expected 1`] = `
<div>
  Authorized
</div>
`;
